.env.example

```diff
@@ -1,3 +0,0 @@
-FLASK_ENV=development
-SECRET_KEY=lkasjdf09ajsdkfljalsiorj12n3490re9485309irefvn,u90818734902139489230
-DATABASE_URL=postgresql://loot_locker_dev@localhost/loot_locker
```

Comment:
The `.env.example` file should be kept as a way to inform developers what their `.env` should look like